src/scss/App.scss

```diff
@@ -52,22 +74,23 @@ body {
     background: $background-color-purple;
 
     .logo-container {
-      display: flex;
-      border-radius: 24px;
-      box-shadow: 1px 1px 5px $border-dark-grey;
-      margin: 0 15px;
-      background-color: white;
+
+      img {
+        display: block;
+        margin: 0 10px;
+        height: 40px;
+      }
     }
 
     button {
-      padding: 0 15px;
+      padding: 0 15px 0 0;
```

Comment:
Let's leave this as `padding: 0 15px;`
and make the img only have `margin-left: 10px;` (see comment above)